onnxruntime/test/framework/cuda/fence_cuda_test.cc

```diff
@@ -1,5 +1,6 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+#if 0  // TODO: Can't call these directly from external code as Cuda is now a shared library
```

Comment:
How do we re-enable these tests? I've not seen other files, but a bit concerned that we're disabling these tests.

Reply:
I need to expose all of the internal methods they call through the ProviderInfo_CUDA() interface so that they can still access them. It's at least another couple weeks of work and I need to do it for other providers also.

The ProviderInfo_CUDA() style interface was created to solve this problem, it's just not yet converted over as it wasn't trivial to change.

Reply:
Is there an easy way to grep for the tests that have been disabled due to this change? This will make it easier to find these tests and quantify.